curves/src/edwards_bls12/fr.rs

```diff
@@ -33,7 +33,7 @@ impl Fp256Parameters for FrParameters {}
 impl FftParameters for FrParameters {
     type BigInteger = BigInteger;
 
-    const POWERS_OF_G: &'static [BigInteger] = &[];
+    const POWERS_OF_ROOTS_OF_UNITY: &'static [BigInteger] = unimplemented!();
```

Comment:
Didn't we want to make this an `Option`?

Reply:
Let's revisit this. I think it subtly complicates the `sqrt` logic, and without a fallback `sqrt` algorithm, the `unimplemented!()` likely suffices.